validate/main.go

```diff
@@ -32,7 +32,7 @@ var (
 func init() {
 	flag.BoolVar(&flagVersion, "version", false, "print the version of ignition-validate")
 	flag.Usage = func() {
-		fmt.Fprintf(flag.CommandLine.Output(), "Usage:\n  %s [flags] config.ign\n\n", os.Args[0])
+		_, _ = fmt.Fprintf(flag.CommandLine.Output(), "Usage:\n  %s [flags] config.ign\n\n", os.Args[0])
```

Comment:
validate: ignore printf errors db9327eddfcd53b19ae52f1342537b0fc235cf5a could have details added to commit message listing lint like other commits, not a big deal but would be cool.